core/cap-0046-06.md

```diff
@@ -171,15 +171,15 @@ fn allowance(e: &Host, from: Address, spender: Address) -> Result<i128, HostErro
 /// * `spender` - The address being authorized to spend the tokens held by
 ///   `from`.
 /// * `amount` - The tokens to be made availabe to `spender`.
-/// * `expiration_ledger` - The ledger number where this allowance expires. Cannot
+/// * `live_until_ledger` - The ledger number where this allowance expires. Cannot
```

Comment:
This isn't an event, but noticed the parameter is incorrectly named vs the contract.